nvme-print.c

```diff
@@ -35,6 +32,8 @@ static struct print_ops *nvme_print_ops(enum nvme_print_flags flags)
 
 	if (flags & JSON)
 		ops = nvme_get_json_print_ops(flags);
+	if (!(flags & (JSON | BINARY)))
```

Comment:
By changing the `if` to `else if` the `JSON` flag condition checking seems to be delete as below.
```c
    else if (!(flags & BINARY))
```